lib/object_store/app/controllers/object_store/content_blocks_controller.rb

```diff
@@ -1,5 +1,58 @@
 class ObjectStore::ContentBlocksController < ApplicationController
+  include ObjectStore::Engine.routes.url_helpers
+
+  before_action :set_content_block, only: %i[show update destroy]
+
+  def info
+    block_type = params.require(:block_type)
+    @schema = ObjectStore::ContentBlockValidator.schema_for(block_type)
+    render json: @schema
+  end
+
   def index
-    render :index
+    @content_blocks = ObjectStore::ContentBlock.all
+  end
+
+  def show
+    render json: @content_block
+  end
+
+  def new
+    block_type = params.require(:block_type)
+
+    @schema = ObjectStore::ContentBlockValidator.schema_for(block_type)
+
+    ActionController::Parameters.action_on_unpermitted_parameters = false
```

Comment:
What does this do here?

Reply:
ah this is a temporary workaround as the next line to get the `properties` params threw an error because it's missing `block_type` (`action_on_unpermitted_parameters` is set to error elsewhere)... will tidy up